src/inc/defines.php

```diff
@@ -249,29 +249,29 @@ public static function getConfigType($config) {
   public static function getConfigDescription($config) {
     switch ($config) {
       case DConfig::BENCHMARK_TIME:
-        return "Time in seconds an agent should benchmark a task";
+        return "How long an agent should benchmark a task (in seconds)";
       case DConfig::CHUNK_DURATION:
-        return "How long an agent approximately should need for completing one chunk";
+        return "Estimate of the time it will take for an agent to complete one chunk of work (in seconds)";
       case DConfig::CHUNK_TIMEOUT:
-        return "How long an agent must not respond until it is treated as timed out and the chunk will get shipped to other agents";
+        return "How long an agent must not respond until it is treated as timed out and the chunk will get shipped to other agents (in seconds)";
       case DConfig::AGENT_TIMEOUT:
-        return "How long an agent must not respond until he is treated as not active anymore";
+        return "How long an agent must not respond until it is treated as no longer active (in seconds)";
       case DConfig::HASHES_PAGE_SIZE:
-        return "How many hashes are showed at once in the hashes view (Page size)";
+        return "How many hashes to display at once in the hashes view (page size)";
       case DConfig::FIELD_SEPARATOR:
-        return "What separator should be used to separate hash and plain (or salt)";
+        return "Separator to use to separate the hash and plaintext (or salt)";
```

Comment:
Reword to something like "Separator used to separate the hash and plaintext (or salt)